rewrite-java/src/main/java/org/openrewrite/java/tree/Space.java

```diff
@@ -286,13 +286,17 @@ public String toString() {
                     printedWs.append(spaces[(i - lastNewline) % 10]);
                 } else if (c == '\t') {
                     printedWs.append(tabs[(i - lastNewline) % 10]);
+                } else {
+                    // should never happen (probably a bug in the parser)
+                    printedWs.append(c);
                 }
             }
         }
+        String whitespaces = printedWs.toString();
 
         return "Space(" +
-               "comments=<" + (comments.size() == 1 ? "1 comment" : comments.size() + " comments") +
-               ">, whitespace='" + printedWs + "')";
```

Comment:
Do agree that looking at `whitespace='')"` can be very confusing
The first are two single quotes